configuration/index.md

```diff
@@ -7,151 +7,151 @@ title: Configuration
+* [Bindings]({{base}}/addons/bindings.html) - The numerous Add-ons to communicate with your devices
+* [Things](things.html) - Your devices represented in openHAB
+* [Items](items.html) - properties and capabilities of your Things
+* [Groups](items.html#groups) - collections or categories containing Items
+* [Sitemaps](sitemaps.html) - User-defined frontend interfaces to arrange Groups, Items and more
+* [Transformations](transform.html) - Helper functions to transform your data
+* [Persistence](persistence.html) - Services to store data over time
+* [Rules](rules-dsl.html) - Automation logic, the heart of your SmartHome!
 
-The individual articles hold all details needed to understand the concepts behind these building bricks for your Smart Home. <!-- Draft -->
-For more details on the base concept behind openHAB, please visit the [Concepts Overview page]({{base}}/concepts/index.html). <!-- Draft -->
+The individual articles hold all details needed to understand the concepts behind these building bricks for your Smart Home.
+For more details on the base concept behind openHAB, please visit the [Concepts Overview page]({{base}}/concepts/index.html).
 
 ## Versatility
 
-openHAB provides different user interfaces to modify settings, manage your things and items, build rules and access the sitemap results. <!-- Draft -->
-Every interface has it's own strong points. <!-- Draft -->
+openHAB provides different user interfaces to modify settings, manage your things and items, build rules and access the sitemap results.
+Every interface has it's own strong points.
 
 ## Comparison
 
-<table class="centered highlight"> <!-- Draft -->
+<table class="centered highlight">
   <thead>
-    <tr> <!-- Draft -->
+    <tr>
       <th data-field="task">Task</th>
       <th data-field="files">Textual Configuration</th>
       <th data-field="paperui">Paper UI</th>
       <th data-field="habmin">HABmin</th>
       <th data-field="uis">other UIs</th>
       <th data-field="karaf">Karaf Console</th>
     </tr>
-  </thead> <!-- Draft -->
+  </thead>
 
   <tbody>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Auto-Discover Things and Items</td>
       <td>❌</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>❌</td>
       <td>✔️</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Define Things</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>❌</td>
       <td>✔️</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Define Items</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>❌</td>
       <td>✔️</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Define Groups</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>✔️</td>
       <td>❌</td>
       <td>✔️</td>
     </tr>
-    <tr> <!-- Draft -->
+    <tr>
       <td>Define Sitemaps</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
       <td>(HABPanel)</td>
```

Comment:
Sorry, indeed no it cannot do it :)

To be exhaustive, the terminology currently used in HABPanel is:
At the top you have a "panel registry", stored as a JSON object inside a configuration property which can contain several "panel configurations", each being a set of "dashboards" (and other settings, widget definitions etc.)... and each dashboard contains "widgets" :)

So I guess you could say it defines and accesses "panel configurations" or "dashboards", but this concept is currently specific to HABPanel, and cannot be easily defined or accessed outside of it; though as stated above the panel registry is a configuration variable so there are ways to alter it using Paper UI (_Configuration > Services > UI > HABPanel > Show more_) or the `config` subshell of the Karaf commandline (`config:edit org.openhab.habpanel` and so on), but it's definitely not recommended.